src/site/content/en/accessible/color-and-contrast/index.md

```diff
@@ -0,0 +1,153 @@
+About 1 in 12 men and 1 in 200 women have some form of "color blindness";
+that means about 1/20th, or 5%, of your users will not experience your site the way you intended.
+When we rely on color to convey information, we push that number to unacceptable levels.
+
+{% Aside %}
+Note: The term "color blindness" is often used to describe a visual condition where a person has trouble distinguishing colors,
+but in fact very few people are truly color blind.
+Most people with color deficiencies can see some or most colors,
+but have difficulty differentiating between certain colors such as reds and greens (most common),
+browns and oranges, and blues and purples.
+{% endAside %}
+
+For example, in an input form,
+a telephone number might be underlined in red to show that it is invalid.
+But to a color deficient or screen reader user, that information is not conveyed well, if at all.
+Thus, you should always try to provide multiple avenues for the user to access critical information.
+
+<figure class="w-figure">
+  <img src="./input-form1.png" alt="A image of an input form with an incorrect phone number highlighted only with a red color.">
+</figure>
+
+The [WebAIM checklist states in section 1.4.1](https://webaim.org/standards/wcag/checklist#sc1.4.1) that
+"color should not be used as the sole method of conveying content or distinguishing visual elements."
+It also notes that "color alone should not be used to distinguish links from surrounding text"
+unless they meet certain contrast requirements.
+Instead, the checklist recommends adding an additional indicator such as an underscore
+(using the CSS text-decoration property) to indicate when the link is active.
+
+An easy way to fix the previous example is to add an additional message to the field,
+ announcing that it is invalid and why.
+
+<figure class="w-figure">
+  <img src="./input-form2.png" alt="The same input form as in the last example, this time with a text label indicating the problem with the field.">
+</figure>
+
+When you're building an app, keep these sorts of things in mind
+and watch out for areas where you may be relying too heavily on color to convey important information.
+
+If you're curious about how your site looks to different people,
+or if you rely heavily on the use of color in your UI,
+you can use DevTools to simulate various forms of visual impairment,
+including different types of color blindness.
+Chrome includes an Emulate Vision Deficiencies feature.
+To access it open DevTools and select Rendering,
+you can then emulate the following color deficiencies.
+
+- Protanopia: the inability to perceive any red light.
+- Deuteranopia: the inability to perceive any green light.
+- Tritanopia: the inability to perceive any blue light.
+- Achromatopsia: the inability to perceive any color except for shades of grey (extremely rare).
+
+<figure class="w-figure">
+  <img src="./emulate.jpg" alt="Emulating the vision of a person with Achromatopsia shows our page in greyscale.">
```

Comment:
I found this screenshot a bit hard to read. Might try retaking it and using cmd-+ to boost the text size in devtools?

Reply:
You can use this:

![image](https://user-images.githubusercontent.com/4713486/83207985-5b7e4180-a109-11ea-8a1f-2aa83948680c.png)

Reply:
```suggestion
  <img class="w-screenshot--filled" src="./emulate.jpg" alt="Emulating the vision of a person with Achromatopsia shows our page in greyscale.">
```